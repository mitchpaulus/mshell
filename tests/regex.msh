"hello, world world" '^h' reMatch ('Found match') ('No match') iff wl
"hello, world world" '^a' reMatch ('Found match') ('No match') iff wl

# Examples from: https://pkg.go.dev/regexp#example-Regexp.ReplaceAllString
"-ab-axxb-" 'a(x*)b' 'T' reReplace wl
"-ab-axxb-" 'a(x*)b' '$1' reReplace wl
"-ab-axxb-" 'a(x*)b' '$1W' reReplace wl
"-ab-axxb-" 'a(x*)b' '${1}W' reReplace wl
