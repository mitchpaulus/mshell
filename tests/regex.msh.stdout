Found match
No match
-T-T-
--xx-
---
-W-xxW-
